### PR TITLE
babel yay

### PR DESCRIPTION
**Pull request by callumacrae**

### CODE REVIEW

File: tasks/browserify.js [hunk 1]
```diff
@@ -29,6 +30,11 @@ module.exports = function (gulp, plugins, options) {
       ignore: ['jquery']
     });
 
+    // The ignore is for the jQuery stream below
+    bundler.transform(babelify.configure({
+      ignore: /_stream_0\.js/
```
Comment (mcmillan) [inline tasks/browserify.js]: We should probably add either a `blacklist` or `whitelist` option here to ensure we only use transformations that don't blow up in IE8.

Comment (mcmillan) [inline tasks/browserify.js]: ...aaaaand I'm gonna stop commenting on WIP code

Comment (callumacrae) [inline tasks/browserify.js]: Didn't know that existed, I'll do that tomorrow. Also need to switch to ESLint—JSHint doesn't support ES6 in the way that we need.

Comment (callumacrae) [inline tasks/browserify.js]: This code isn't WIP, there's just other things to add to the PR.

**Pull request merged**